modules/cco-determine-mode-gui.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-within-minor.adoc
+// * authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc
+
+:_content-type: PROCEDURE
+
+ifeval::["{context}" == "updating-cluster-within-minor"]
+:console:
+endif::[]
+ifeval::["{context}" == "about-cloud-credential-operator"]
+:about-cco:
+endif::[]
+
+[id="cco-determine-mode-gui_{context}"]
+= Determining the Cloud Credential Operator mode by using the web console
+
+You can determine what mode the Cloud Credential Operator (CCO) is configured to use by using the web console.
+
+[NOTE]
+====
+Only Amazon Web Services (AWS), global Microsoft Azure, and Google Cloud Platform (GCP) clusters support multiple CCO modes.
+====
+
+.Prerequisites
+
+* You have access to an {product-title} account with cluster administrator permissions.
+
+.Procedure
+
+. Log in to the {product-title} web console as a user with the `cluster-admin` role.
+
+. Navigate to *Administration* -> *Cluster Settings*.
+
+. On the *Cluster Settings* page, select the *Configuration* tab.
+
+. Under *Configuration resource*, select *CloudCredential*.
+
+. On the *CloudCredential details* page, select the *YAML* tab.
+
+. In the YAML block, check the value of `spec.credentialsMode`. The following values are possible, though not all are supported on all platforms:
++
+--
+* `''`: The CCO is operating in the default mode. In this configuration, the CCO operates in mint or passthrough mode, depending on the credentials provided during installation.
```

Comment:
For the non-upgrade case, I want to add a step here to show how to look at the annotation and determine which it is.